src/ngraph/op/fused/lstm_cell.cpp

```diff
@@ -274,38 +266,38 @@ NodeVector op::LSTMCell::decompose_op() const
     return {H, C};
 }
 
-Output<Node> op::LSTMCell::get_bias() const
-{
-    Output<Node> bias;
-    // Split B onto Wb an Rb and add them.
-    NodeVector b_W_R = builder::split(input_value(5), 2);
-    bias = b_W_R.at(0) + b_W_R.at(1);
-    return bias;
-}
-
-NodeVector op::LSTMCell::get_peephole_weights() const
+Output<Node> op::LSTMCell::get_default_bias_input() const
 {
-    Output<Node> P;
-    P = input_value(6);
-    return builder::split(P, s_peepholes_count);
+    return Output<Node>{
+        op::Constant::create(input(0).get_element_type(),
+                             Shape{s_gates_count * get_hidden_size()},
+                             vector<float>(s_gates_count * get_hidden_size(), 0.f))};
 }
 
-void op::LSTMCell::add_default_bias_input()
+Output<Node> op::LSTMCell::get_default_peepholes_input() const
 {
-    Output<Node> B =
+    return Output<Node>{
         op::Constant::create(input(0).get_element_type(),
-                             Shape{2 * s_gates_count * get_hidden_size()},
-                             vector<float>(2 * s_gates_count * get_hidden_size(), 0.f));
-    set_argument(5, B);
+                             Shape{s_peepholes_count * get_hidden_size()},
+                             vector<float>(s_peepholes_count * get_hidden_size(), 0.f))};
 }
 
-void op::LSTMCell::add_default_peepholes_input()
+shared_ptr<Node> ngraph::op::LSTMCell::convert_node_format(const Output<Node>& node) const
 {
-    Output<Node> P =
-        op::Constant::create(input(0).get_element_type(),
-                             Shape{s_peepholes_count * get_hidden_size()},
-                             vector<float>(s_peepholes_count * get_hidden_size(), 0.f));
-    set_argument(6, P);
+    static const std::map<op::LSTMWeightsFormat, std::vector<size_t>> gate_order_conversion_map{
+        {op::LSTMWeightsFormat::FICO, {1, 0, 2, 3}},
+        {op::LSTMWeightsFormat::ICOF, {0, 3, 1, 2}},
+        {op::LSTMWeightsFormat::IFOC, {0, 1, 3, 2}},
+        {op::LSTMWeightsFormat::IOFC, {0, 2, 3, 1}},
```

Comment:
We're not missing it, since we're converting other supported formats to `IFCO` as it is in [function doc](https://github.com/NervanaSystems/ngraph/blob/tsocha/fused-lstm/src/ngraph/op/fused/lstm_cell.hpp#L169-L178).